### ISSUE TITLE
Need Subversion scm plugin.

### ISSUE THREAD

**Issue by sfloess** [solenopsis/Solenopsis#58]
Once issue #38  is done, need to be able to plugin Subversion as an scm client.

**Comment by sfloess** [comment]
Closing as superseded by #286 (Implement Subversion SCM plugin as proof-of-concept), which is part of the proper decomposition of the SCM plugin architecture (#38, #282, #284, #285, #286). The newer issues provide a more structured approach with a defined plugin contract.
